website/src/components/header/header.astro

```diff
@@ -194,6 +194,7 @@ const { noDrawer } = Astro.props;
           <li><Link class="item link" href="/tooling">Tooling support</Link></li>
         </ul>
       </details>
+      <Link class="item link" href="/videos">Videos</Link>
```

Comment:
videos after docs?

Reply:
Yes, trying to boost website engagement

Reply:
Yes, trying to boost website engagement so I want the Videos page to be prominently listed.

Reply:
Oh, I see what you mean, I did intend to have Videos after Docs so I made that change.